typescript/lib/config.ts

```diff
@@ -409,6 +411,10 @@ export class AIConfigRuntime implements AIConfig {
     return result;
   }
 
+  public setCallbackManager(callbackManager: CallbackManager) {
```

Comment:
In which scenarios would someone construct the config and then pass the CM afterwards, vs just supporting it passed through in the constructor?

Also, seems like this should make some sort of noise if there's already a CM that would be replaced by this call?

Reply:
The idea here is that you can modify, delete the callback manager with one command. If it might should it throw an error or do something else?

Reply:
Maybe just check and do a `console.warn('Replaced existing CallbackManager for config')`?